curp/Cargo.toml

```diff
@@ -34,6 +34,8 @@ madsim = { version = "0.2.22", features = ["rpc", "macros"] }
 opentelemetry = "0.18.0"
 parking_lot = "0.12.1"
 prost = "0.11"
+prost-wkt = "0.4"
+prost-wkt-types = "0.4"
```

Comment:
These dependencies are not used.

Reply:
It seems compiling the `google.protobuf.Empty` indeed does not require these deps. Removed.